content/doc/developer/security/xss-prevention.adoc

```diff
@@ -0,0 +1,98 @@
+---
+title: Preventing Cross-Site Scripting in Jelly views
```

Comment:
The page just says `xss-prevention`. Maybe it makes sense to keep placeholders for Groovy and Javascript XSS prevention